docs/api-reference/supervisor-api.md

```diff
@@ -23,89 +23,3075 @@ sidebar_label: Supervisors
+                "startDelay": "PT5S",
+                "period": "PT30S",
+                "useEarliestOffset": true,
+                "completionTimeout": "PT1800S",
+                "lateMessageRejectionPeriod": null,
+                "earlyMessageRejectionPeriod": null,
+                "lateMessageRejectionStartDateTime": null,
+                "configOverrides": null,
+                "idleConfig": null,
+                "stream": "social_media",
+                "useEarliestSequenceNumber": true
+            },
+            "context": null,
+            "suspended": false
+        },
+        "version": "2023-07-05T20:59:16.872Z"
+    }
+]
+  ```
+</details>
+
+## Managing Supervisors
 
-`GET /druid/indexer/v1/supervisor/<supervisorId>`
+### Create a new Supervisor or update an existing Supervisor
 
-Returns the current spec for the supervisor with the provided ID.
+#### URL
 
-`GET /druid/indexer/v1/supervisor/<supervisorId>/status`
+<code className="postAPI">POST</code> `/druid/indexer/v1/supervisor`
 
-Returns the current status of the supervisor with the provided ID.
+Creates a new Supervisor or updates an existing one for the same datasource with a new schema and configuration. 
 
-`GET/druid/indexer/v1/supervisor/history`
+You can create a supervisor spec using [Apache Kafka](../development/extensions-core/kafka-ingestion.md#define-a-supervisor-spec) or [Amazon Kinesis](../development/extensions-core/kinesis-ingestion.md#submitting-a-supervisor-spec). Once created, a Supervisor will persist in the metadata database.
 
-Returns an audit history of specs for all supervisors (current and past).
+Using this endpoint on an existing Supervisor for the same datasource will result in the running Supervisor signaling its tasks to stop reading and begin publishing, exiting itself, and a new Supervisor being created with the provided configuration from the request body. This submits a new schema while retaining existing publishing tasks and starting new tasks at the previous task offsets. 
 
-`GET /druid/indexer/v1/supervisor/<supervisorId>/history`
+#### Responses
 
-Returns an audit history of specs for the supervisor with the provided ID.
+<!--DOCUSAURUS_CODE_TABS-->
 
-`POST /druid/indexer/v1/supervisor`
+<!--200 SUCCESS-->
+<br/>
+*Successfully created a new Supervisor or updated an existing Supervisor* 
+<!--415 UNSUPPORTED MEDIA TYPE-->
+<br/>
+*Request body content type is not in JSON format* 
 
-Create a new supervisor or update an existing one.
+<!--END_DOCUSAURUS_CODE_TABS-->
 
-`POST /druid/indexer/v1/supervisor/<supervisorId>/suspend`
+---
+
+#### Sample request
+
+The following example uses JSON input format to create a supervisor spec for Kafka with a `social_media` datasource and `social_media` topic. 
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl "http://<ROUTER_IP>:<ROUTER_PORT>/druid/indexer/v1/supervisor" \
+--header "Content-Type: application/json" \
+--data "{
+    \"type\": \"kafka\",
```

Comment:
Can you update this example so that it doesn't escape JSON characters? This makes it hard to read. (unless escaping is required)